http://bugs.gentoo.org/show_bug.cgi?id=299295

since we can't run autoreconf here, patching only configure is ok

--- configure
+++ configure
@@ -7351,6 +7351,10 @@
     lt_cv_sys_max_cmd_len=8192;
     ;;
 
+  mint*)
+    lt_cv_sys_max_cmd_len=8192;
+    ;;
+
   amigaos*)
     # On AmigaOS with pdksh, this test takes hours, literally.
     # So we just punt and use a minimum line length of 8192.
